sentry_sdk/integrations/opentelemetry/integration.py

```diff
@@ -0,0 +1,163 @@
+"""
+IMPORTANT: The contents of this file are part of a proof of concept and as such
+are experimental and not suitable for production use. They may be changed or
+removed at any time without prior notice.
+"""
+import sys
+from importlib import import_module
+
+from sentry_sdk.integrations import DidNotEnable, Integration
+from sentry_sdk.integrations.opentelemetry.span_processor import SentrySpanProcessor
+from sentry_sdk.integrations.opentelemetry.propagator import SentryPropagator
+from sentry_sdk.integrations.modules import _get_installed_modules
+from sentry_sdk.utils import logger
+from sentry_sdk._types import TYPE_CHECKING
+
+try:
+    from opentelemetry import trace  # type: ignore
+    from opentelemetry.instrumentation.auto_instrumentation._load import (  # type: ignore
+        _load_distro,
+        _load_instrumentors,
+    )
+    from opentelemetry.propagate import set_global_textmap  # type: ignore
+    from opentelemetry.sdk.trace import TracerProvider  # type: ignore
+except ImportError:
+    raise DidNotEnable("opentelemetry not installed")
+
+if TYPE_CHECKING:
+    from typing import Dict
+
+
+INSTRUMENTED_CLASSES = {
+    # A mapping of packages to (original class, instrumented class) pairs. This
+    # is used to post-instrument any classes that were imported before OTel
+    # instrumentation took place.
+    "fastapi": (
+        "fastapi.FastAPI",
+        "opentelemetry.instrumentation.fastapi._InstrumentedFastAPI",
+    ),
+    "flask": ("flask.Flask", "opentelemetry.instrumentation.flask._InstrumentedFlask"),
+}
+
+
+class OpenTelemetryIntegration(Integration):
+    identifier = "opentelemetry"
+
+    @staticmethod
+    def setup_once():
+        # type: () -> None
+        logger.warning(
+            "[OTel] Initializing highly experimental OpenTelemetry support. Use at your own risk."
+        )
+
+        original_classes = _record_unpatched_classes()
+
+        try:
+            distro = _load_distro()
```

Comment:
can we make sure we document the differences to https://github.com/open-telemetry/opentelemetry-python-contrib/blob/7603a1fc69474398289c2944796249e70bba0c82/opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/sitecustomize.py#L29-L41 in the PR description.


For instance
* we're not running `load_configurators`
* we're not replacing `PYTHON_PATH`

does this anyway impact any of otel's instrumentation further on?

Reply:
will do